github4s/shared/src/main/scala/github4s/HttpClient.scala

```diff
@@ -80,8 +80,8 @@ class HttpRequestBuilder[C, M[_]](
 
   def withAuth(accessToken: Option[String] = None) = {
     val authHeader = accessToken match {
-      case Some(token) ⇒ Map("Authorization" → s"token $token")
-      case _           ⇒ Map.empty[String, String]
+      case Some(token) => Map("Authorization" → s"token $token")
```

Comment:
should we to take care of `→` as well? maybe in a separate PR

Reply:
Was planning on a separate PR, i'll get a ticket started! Thanks